tests/explorer.api.tests/QueryTests.cs

```diff
@@ -169,6 +170,37 @@ public async void TestRepeatingRows()
             });
         }
 
+        [Fact]
+        public async void TestCancelQuery()
+        {
+            using var client = new HttpClient() { BaseAddress = TestWebAppFactory.Config.AircloakApiUrl };
+            var authProvider = factory.EnvironmentVariableAuthProvider();
+            var pollFrequency = TimeSpan.FromMilliseconds(10);
+            var jsonApiClient = new JsonApiClient(client, authProvider);
+            var query = new Min("loans", "amount", null);
+
+            using var cts = new CancellationTokenSource();
+            var queryInfo = await jsonApiClient.SubmitQuery("gda_banking", query.QueryStatement, cts.Token);
+            cts.CancelAfter(1);
```

Comment:
This can simply be `cts.Cancel()`

Reply:
fixed with 1291bc2d3824a5c765573950679d8817613a7754